framework_tvb/tvb/core/entities/file/files_helper.py

```diff
@@ -43,6 +43,7 @@
 from tvb.core.entities.file.exceptions import FileStructureException
 from tvb.core.entities.file.xml_metadata_handlers import XMLReader, XMLWriter
 from tvb.core.entities.transient.structure_entities import GenericMetaData
+from tvb.core.data_encryption_handler import DataEncryptionHandler
```

Comment:
i understand the reasons for putting this here, but as part of another task we identified that this `helper` (already a not so nice pattern) has become much too heavy.
so, could you maybe move the renaming and removal of the enc folders in the upper layers (the services where for sire these methods are called)?